psydac/linalg/basic.py

```diff
@@ -52,6 +59,11 @@ def shape( self ):
     def space( self ):
         pass
 
+    @property
+    @abstractmethod
+    def dtype( self ):
+        pass
+
```

Comment:
My original comment was

> What is this `dtype` property?
> Why is it not part of the abstract interface of the `VectorSpace` base class?

So I was expecting you to answer my questions in writing, explaining why you need that property and where it is used. I did not say "Please add `dtype` to `VectorSpace", did I?

So I think that we should discuss about this.

cc: @campospinto

Reply:
okay we can discuss about it